components/backup-plugins/README.md

```diff
@@ -0,0 +1,57 @@
+# Backup plugins
+
+## Overview
+
+Backup plugins provide an implementation of a few plugins required to properly restore the Kyma cluster and resources created inside it using Velero. They focus mainly on resources related to the Service Catalog, such as instances or bindings. Each plugin is defined in a separate file inside the [`internal/plugins`](internal/plugins) folder. The purpose of each plugin is defined in the file's comments.
+
+The structure of folders and files is based on the [Velero plugin example repository](https://github.com/heptio/velero-plugin-example).
+
+## Installation
+
+Run a Docker image build from the [Dockerfile](Dockerfile) as an init container inside the deployment of the [Velero chart](../../resources/velero). During development, you can push a built image to your own image repository and run:
+
+```bash
+velero plugin add {yourRepo/imageName:tag}
+```
+
+## Create a new plugin
+
+- New plugins must be added under [`internal/plugins`](internal/plugins) :
+
+```             
+  ├── internal                                                                  
+    ├── plugins
+      ├── backup    # new Backup Item Action plugins 
+      ├── restore   # new Restore Item Action plugins 
+  ```
+
+- All new plugins must be registered in the `components/backup-plugins/main.go` file.
+- **Backup Item Action** - performs arbitrary logic on individual items prior to storing them in the backup file.
+- **Restore Item Action** - performs arbitrary logic on individual items prior to restoring them in the Kyma cluster.
```

Comment:
I'm wondering if we should leave these two here or move them to the overview section. They can be left here, but then maybe we need a kind of introduction, such as:

This folder includes the `backup` and `restore` subfolders, where you can define the following plugins:

Question: is the `Backup Item Action/ Restore Item Action` an object?